appconfiguration/azure-spring-cloud-feature-management/dynamic-features-sample/README.md

```diff
@@ -0,0 +1,47 @@
+# Feature Variant Sample
```

Comment:
Does this sample require an App Configuration instance?

Reply:
No